src/rgw/rgw_sal_motr.cc

```diff
@@ -1257,8 +1257,49 @@ MotrObject::~MotrObject() {
 
 int MotrObject::set_obj_attrs(const DoutPrefixProvider* dpp, RGWObjectCtx* rctx, Attrs* setattrs, Attrs* delattrs, optional_yield y, rgw_obj* target_obj)
 {
-  // TODO: implement
-  ldpp_dout(dpp, 20) <<__func__<< ": MotrObject::set_obj_attrs()" << dendl;
+  if (this->category == RGWObjCategory::MultiMeta)
```

Comment:
What about multipart objects. why are we not setting object tags for it.
This was skipped as part of POC code but we need to implement it for multipart objects as well.

This has to be done for get_obj_attrs api as well.

Reply:
Hi @sjain09 , Added a comment as TODO.. We already have a ticket to enable tagging for multipart objects. 
cc: @cdeshmukh

Reply:
yes we have https://jts.seagate.com/browse/CORTX-30353 for same

@selvakumaar5496 add a TODO comment in code also, i dont see it still. dont add ticket number there

Reply:
ok